### ISSUE TITLE
📚 Make a beautiful README.md

### ISSUE THREAD

**Issue by Salz0** [Salz0/telegram_flea#18]
The README.md right now is rather ordinary and deserves some refreshment. 

Provide a nice new version of the first page of our project

- [ ] I suggest adding a GIF demonstration on how the bot works, would be really nice. 
- [x] Also broaden the table of languages, adding the top 20 most popular languages for us to consider in the future
- [x] Rephrase and delete unnecessary wording
- [x] Add useful link (f.e. contributing guide)

**Comment by sfazli96** [comment]
Hi, I am a first-time GitHub contributor, Can I give this issue a try or another issue? Thank you!

**Comment by sfazli96** [comment]
Just wanted to update, I am working on: Also broaden the table of languages, adding the top 20 most popular languages for us to consider in the future
 Rephrase and delete unnecessary wording
I have added more popular language. I am still working on the gif part. Will push to forked branch first, when its done.

**Comment by sfazli96** [comment]
Cool, I just created the 1st pull request, waiting on how it is.
